Bottom: f4b8a401e1fe7aebce62c3921ae66a120fd03d4f
Top:    72e1213f2c7fddbc77d6ed3c03202d017d18eac5
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-21 17:15:28 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 6911421..e06c3ea 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -76,16 +76,16 @@
 #include <asm/uaccess.h>
 #include <linux/interrupt.h>
 
+int rx_batched = 1;
+module_param(rx_batched, int, 0444);
+MODULE_PARM_DESC(rx_batched, "Number of packets batched during rx");
+
 /* Uncomment to enable debugging */
 /* #define TUN_DEBUG 1 */
 
 #ifdef TUN_DEBUG
 static int debug;
 
-static int rx_batched = 1;
-module_param(rx_batched, int, 0444);
-MODULE_PARM_DESC(rx_batched, "Number of packets batched during rx");
-
 #define tun_debug(level, tun, fmt, args...)			\
 do {								\
 	if (tun->debug)						\
